File: Frontend/farm-ui/src/app/entities/Base.Entity.ts
export class BaseEntity {
    createdAt!: Date | null;
    createdById!: any | null;
    updatedAt!: Date | null;
    updatedById!: any | null;
    deletedAt!: Date | null;
    deletedById!: any | null;
    isSoftDeleted!: any | null;
}

File: API/FarmProductionAPI.Domain/Models/BaseEntity.cs
namespace FarmProductionAPI.Domain.Models
{
    public class BaseEntity
    {
        public Guid? Id { get; set; }
        public Guid? CreatedById { get; set; }
        public Guid? UpdatedById { get; set; }
        public Guid? DeletedById { get; set; }
        public DateTime? CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
        public DateTime? DeletedAt { get; set; }
        public bool? IsSoftDeleted { get; set; }
    }
}
